Update the IGX Thor devkit MB1 pinmux and GPIO BCT to NVIDIA's r38.5 version

r39.2.1 ships the IGX (p3834-0008 + p3740-0002) pinmux and GPIO BCT generated
from pinmux spreadsheet revision 14, byte-identical to r38.2.1. r38.4 and
r38.5 (IGX-SW 2.0, NVIDIA's supported IGX Thor release) ship revision 18,
which configures the DP AUX pads, parks PEX L3 CLKREQ#, muxes SGMII1 MDIO/MDC,
and enables the FSI GPIO inputs AG5/AG6/AH2/AH4. The files below are copied
verbatim from the r38.5 BSP. The pad voltage BCT differs only in comments.

diff --git a/bootloader/tegra264-mb1-bct-gpio-p3834-0008-p3740-0002.dtsi b/bootloader/tegra264-mb1-bct-gpio-p3834-0008-p3740-0002.dtsi
index 8428b19..a8b54db 100644
--- a/bootloader/tegra264-mb1-bct-gpio-p3834-0008-p3740-0002.dtsi
+++ b/bootloader/tegra264-mb1-bct-gpio-p3834-0008-p3740-0002.dtsi
@@ -1,7 +1,6 @@
-/*This dtsi file was generated by prometheus_p3740_cvm_p3834.xlsm Revision: 14 */
+/*This dtsi file was generated by T264_Pinmux_Config_CVB_Prometheus_P3740.xlsm Revision: 18 */
 /*
- * SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights
- * reserved.
+ * SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
  * SPDX-License-Identifier: BSD-3-Clause
  *
  * Redistribution and use in source and binary forms, with or without
@@ -13,7 +12,7 @@
  * 2. Redistributions in binary form must reproduce the above copyright notice,
  * this list of conditions and the following disclaimer in the documentation
  * and/or other materials provided with the distribution.
- *
+ * 
  * 3. Neither the name of the copyright holder nor the names of its
  * contributors may be used to endorse or promote products derived from
  * this software without specific prior written permission.
@@ -135,6 +134,9 @@ gpio@b0320000 {
 		gpio-input = <
 			TEGRA264_FSI_GPIO(AB, 4)
 			TEGRA264_FSI_GPIO(AC, 3)
+			TEGRA264_FSI_GPIO(AG, 5)
+			TEGRA264_FSI_GPIO(AG, 6)
+			TEGRA264_FSI_GPIO(AH, 4)
 			>;
 		gpio-output-low = <
 			TEGRA264_FSI_GPIO(AC, 2)
diff --git a/bootloader/tegra264-mb1-bct-pinmux-p3834-0008-p3740-0002.dtsi b/bootloader/tegra264-mb1-bct-pinmux-p3834-0008-p3740-0002.dtsi
index 1b0ec48..c590118 100644
--- a/bootloader/tegra264-mb1-bct-pinmux-p3834-0008-p3740-0002.dtsi
+++ b/bootloader/tegra264-mb1-bct-pinmux-p3834-0008-p3740-0002.dtsi
@@ -1,7 +1,6 @@
-/*This dtsi file was generated by prometheus_p3740_cvm_p3834.xlsm Revision: 14 */
+/*This dtsi file was generated by T264_Pinmux_Config_CVB_Prometheus_P3740.xlsm Revision: 18 */
 /*
- * SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES.
- * All rights reserved.
+ * SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
  * SPDX-License-Identifier: BSD-3-Clause
  *
  * Redistribution and use in source and binary forms, with or without
@@ -30,7 +29,6 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-
 #include "t264-pinctrl-tegra.h"
 
 #include "./tegra264-mb1-bct-gpio-p3834-0008-p3740-0002.dtsi"
@@ -357,17 +355,6 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
-			pex_l3_clkreq_n_pb4 {
-				nvidia,pins = "pex_l3_clkreq_n_pb4";
-				nvidia,function = "pe3_clkreq_l";
-				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
-				nvidia,tristate = <TEGRA_PIN_DISABLE>;
-				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
-				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
-				nvidia,e-io-od = <TEGRA_PIN_ENABLE>;
-				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
-			};
-
 			pex_l3_rst_n_pb5 {
 				nvidia,pins = "pex_l3_rst_n_pb5";
 				nvidia,function = "pe3_rst_l";
@@ -540,6 +527,28 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
+			sgmii1_sma_mdio_paj2 {
+				nvidia,pins = "sgmii1_sma_mdio_paj2";
+				nvidia,function = "sgmii1_sma_mdio";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			sgmii1_sma_mdc_paj3 {
+				nvidia,pins = "sgmii1_sma_mdc_paj3";
+				nvidia,function = "sgmii1_sma_mdc";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
 			can2_dout_pad0 {
 				nvidia,pins = "can2_dout_pad0";
 				nvidia,function = "can2_dout";
@@ -679,6 +688,94 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
+			dp_aux_ch0_p {
+				nvidia,pins = "dp_aux_ch0_p";
+				nvidia,function = "dp_aux_ch0_p";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			dp_aux_ch0_n {
+				nvidia,pins = "dp_aux_ch0_n";
+				nvidia,function = "dp_aux_ch0_n";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			dp_aux_ch1_p {
+				nvidia,pins = "dp_aux_ch1_p";
+				nvidia,function = "i2c10_clk";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_ENABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			dp_aux_ch1_n {
+				nvidia,pins = "dp_aux_ch1_n";
+				nvidia,function = "i2c10_dat";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_ENABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			dp_aux_ch2_p {
+				nvidia,pins = "dp_aux_ch2_p";
+				nvidia,function = "dp_aux_ch2_p";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			dp_aux_ch2_n {
+				nvidia,pins = "dp_aux_ch2_n";
+				nvidia,function = "dp_aux_ch2_n";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			dp_aux_ch3_p {
+				nvidia,pins = "dp_aux_ch3_p";
+				nvidia,function = "dp_aux_ch3_p";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			dp_aux_ch3_n {
+				nvidia,pins = "dp_aux_ch3_n";
+				nvidia,function = "dp_aux_ch3_n";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
 			uart10_tx_ph2 {
 				nvidia,pins = "uart10_tx_ph2";
 				nvidia,function = "uartk_txd";
@@ -1379,9 +1476,31 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
-			sgmii1_sma_mdio_paj2 {
-				nvidia,pins = "sgmii1_sma_mdio_paj2";
-				nvidia,function = "rsvd2";
+			soc_gpio233_pag5 {
+				nvidia,pins = "soc_gpio233_pag5";
+				nvidia,function = "rsvd0";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			soc_gpio234_pag6 {
+				nvidia,pins = "soc_gpio234_pag6";
+				nvidia,function = "rsvd0";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
+			soc_gpio237_pah2 {
+				nvidia,pins = "soc_gpio237_pah2";
+				nvidia,function = "rsvd0";
 				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
 				nvidia,tristate = <TEGRA_PIN_DISABLE>;
 				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
@@ -1390,6 +1509,17 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
+			spi9_sck_pah4 {
+				nvidia,pins = "spi9_sck_pah4";
+				nvidia,function = "rsvd1";
+				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
+				nvidia,tristate = <TEGRA_PIN_DISABLE>;
+				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
 			soc_gpio250_pf0 {
 				nvidia,pins = "soc_gpio250_pf0";
 				nvidia,function = "dca_vsync";
@@ -1994,6 +2124,7 @@ pinmux@ac281000 {
 				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
+
 		};
 
 	unused_lowpower {
@@ -2019,6 +2150,17 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
+			pex_l3_clkreq_n_pb4 {
+				nvidia,pins = "pex_l3_clkreq_n_pb4";
+				nvidia,function = "rsvd1";
+				nvidia,pull = <TEGRA_PIN_PULL_DOWN>;
+				nvidia,tristate = <TEGRA_PIN_ENABLE>;
+				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
+				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
+				nvidia,e-io-od = <TEGRA_PIN_ENABLE>;
+				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
+			};
+
 			soc_gpio113_pb6 {
 				nvidia,pins = "soc_gpio113_pb6";
 				nvidia,function = "rsvd1";
@@ -2194,28 +2336,6 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
-			soc_gpio233_pag5 {
-				nvidia,pins = "soc_gpio233_pag5";
-				nvidia,function = "rsvd0";
-				nvidia,pull = <TEGRA_PIN_PULL_DOWN>;
-				nvidia,tristate = <TEGRA_PIN_ENABLE>;
-				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
-				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
-				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
-				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
-			};
-
-			soc_gpio234_pag6 {
-				nvidia,pins = "soc_gpio234_pag6";
-				nvidia,function = "rsvd0";
-				nvidia,pull = <TEGRA_PIN_PULL_DOWN>;
-				nvidia,tristate = <TEGRA_PIN_ENABLE>;
-				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
-				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
-				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
-				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
-			};
-
 			soc_gpio249_pag7 {
 				nvidia,pins = "soc_gpio249_pag7";
 				nvidia,function = "rsvd0";
@@ -2249,17 +2369,6 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
-			soc_gpio237_pah2 {
-				nvidia,pins = "soc_gpio237_pah2";
-				nvidia,function = "rsvd0";
-				nvidia,pull = <TEGRA_PIN_PULL_DOWN>;
-				nvidia,tristate = <TEGRA_PIN_ENABLE>;
-				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
-				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
-				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
-				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
-			};
-
 			soc_gpio238_pah3 {
 				nvidia,pins = "soc_gpio238_pah3";
 				nvidia,function = "rsvd0";
@@ -2271,17 +2380,6 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
-			spi9_sck_pah4 {
-				nvidia,pins = "spi9_sck_pah4";
-				nvidia,function = "rsvd1";
-				nvidia,pull = <TEGRA_PIN_PULL_DOWN>;
-				nvidia,tristate = <TEGRA_PIN_ENABLE>;
-				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
-				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
-				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
-				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
-			};
-
 			spi9_miso_pah5 {
 				nvidia,pins = "spi9_miso_pah5";
 				nvidia,function = "rsvd1";
@@ -2337,17 +2435,6 @@ pinmux@ac281000 {
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
 
-			sgmii1_sma_mdc_paj3 {
-				nvidia,pins = "sgmii1_sma_mdc_paj3";
-				nvidia,function = "rsvd2";
-				nvidia,pull = <TEGRA_PIN_PULL_DOWN>;
-				nvidia,tristate = <TEGRA_PIN_ENABLE>;
-				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
-				nvidia,drv-type = <TEGRA_PIN_1X_DRIVER>;
-				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
-				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
-			};
-
 			uerror_paj4 {
 				nvidia,pins = "uerror_paj4";
 				nvidia,function = "rsvd1";
@@ -2742,6 +2829,7 @@ pinmux@ac281000 {
 				nvidia,e-io-od = <TEGRA_PIN_DISABLE>;
 				nvidia,e-lpbk = <TEGRA_PIN_DISABLE>;
 			};
+
 		};
 
 		drive {
